kubetest/main.go

```diff
@@ -515,15 +515,26 @@ func prepareGcp(o *options) error {
 	if err := migrateGcpEnvAndOptions(o); err != nil {
 		return err
 	}
-	if o.provider == "gke" && o.gcpNodeImage != "" {
+	if o.provider == "gke" {
+		if o.gcpNodeImage == "" {
+			o.gcpNodeImage = "container_vm"
```

Comment:
for here I'd prefer we set the flag explicitly from config.json, rather than magically have it default to container_vm. Or there are jobs we expect to have an unset `o.gcpNodeImage`?